src/components/Button/BackButton.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { IoIosArrowBack } from "react-icons/io";
+import { withRouter } from 'react-router-dom';
+
+const BackButton = ({history}) => {
+  return (
+    <div
+    onClick={() => history.goBack()}
+      className="regular_16 flex items-center pb-20"
+    >
+      <IoIosArrowBack style={{ marginRight: 20 }} />
```

Comment:
Use the exported icon from the figma document for consistency.

Reply:
Okay done